ui/apps/dev-server-ui/src/app/(dashboard)/functions/config/FunctionConfigurationContainer.tsx

```diff
@@ -17,9 +28,59 @@ export function FunctionConfigurationContainer({ onClose, functionSlug }: Functi
     }
   );
 
+  const [invokeFunction] = useInvokeFunctionMutation();
+
+  const triggers = data?.functionBySlug?.triggers;
+
+  const router = useRouter();
+  const doesFunctionAcceptPayload = useMemo(() => {
+    return Boolean(triggers?.some((trigger) => trigger.type === FunctionTriggerTypes.Event));
+  }, [triggers]);
```

Comment:
What's the reason for using `useMemo`? The number of triggers should be very low

Reply:
It won't matter for child components since it's a primitive. React compares primitives by value when decided to rerender.

The `useMemo` doesn't matter from the child component's perspective. It's just receiving `true` or `false` irrespective of whether the value was memoized